model/cluster_message.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 const (
+	// ClusterMessage Events
```

Comment:
nit: this comment doesn't follow GoDoc conventions. Also, does it add much value?